doc/guides/collaborator-guide.md

```diff
@@ -95,9 +96,8 @@ request must pass code review and CI before landing into the codebase.
 
 ### Code Reviews
 
-At least two Collaborators must approve a pull request before the pull request
-lands. One Collaborator approval is enough if the pull request has been open
-for more than seven days.
+At least one Collaborator must approve a pull request before the pull request
+lands. If possible, waiting for a second approval is strongly recommended.
```

Comment:
"If possible" - useless prefix, it's always possible to wait.

"Strongly recommended" - social pressure language. Most people will read it and think "I better follow this or face disapproval."

I'd just drop the second line altogether.

Reply:
I haven’t dropped the line (but I can do that if you feel strongly about it), but reduced it to “Waiting for a second approval is recommended but not mandatory”. Does that work for you?